examples/authenticated-custom-jwt/app.js

```diff
@@ -24,7 +24,7 @@ app.post("/login", (req, res) => {
     loginDetails.password === mockedPassword
   ) {
     let token = jwt.sign({ username: loginDetails.username }, config.secret, {
-      expiresIn: "24h" // expires in 24 hours
+      expiresIn: "12h" // expires in 12 hours
```

Comment:
Edit: This change is no longer necessary, but it doesn't hurt. I hadn't made my provider yet and was confused.